package.json

```
@@ -19,7 +19,7 @@
     "build": "forge build",
     "build:optimized": "FOUNDRY_PROFILE=optimized forge build",
     "test": "yarn test:unit && yarn test:e2e",
```

Can we change this to:
```suggestion
    "test": "forge test -vvv",
```